applications/dashboard/scss/style-compat.scss

```diff
@@ -1,4 +1,6 @@
 // Compatibility
+@import "compatibility/_mixins.scss";
```

Comment:
This should be named similarly to the other imports.

Reply:
What do you mean? "compatibility_mixins"?